services/QuillLMS/lib/tasks/app_settings.rake

```diff
@@ -0,0 +1,15 @@
+namespace :app_settings do
+  desc 'Creates a new AppSetting.'
+  # Example usage: rake 'app_settings:create[theName,true,true,50,[]]'
```

Comment:
There's a gotcha with zsh that requires rake tasks to be quoted like above when using params